src/Symfony/Component/Routing/Loader/Configurator/Traits/RouteTrait.php

```diff
@@ -74,7 +66,7 @@ final public function utf8(bool $utf8 = true): self
      *
      * @return $this
      */
-    final public function condition(string $condition): self
+    final public function condition(string $condition): static
```

Comment:
seems odd that some methods in this trait have not gotten this change like `host`

Reply:
fixed